all-repos.yaml

```diff
@@ -30,6 +30,7 @@
 - https://github.com/elidupuis/mirrors-jscs
 - https://github.com/elidupuis/mirrors-sass-lint
 - https://github.com/jumanjihouse/pre-commit-hooks
+- https://github.com/jumanjihouse/yamlfmt
```

Comment:
huh this is confusing, there's another tool called `yamlfmt` in python (and it's different from this one?): https://github.com/mmlb/yamlfmt

Reply:
additionally, the package being added here clobbers the `pre_commit_hooks` python namespace (and should pick a different name/layout)

Reply:
should i rename my repo as `pre-commit-hook-yamlfmt` and update this?

do you mean that https://github.com/jumanjihouse/yamlfmt/blob/master/setup.py#L12 clobbers the namespace?
I can fix that as well!

Reply:
yep, both of those sound good

Reply:
updated

Reply:
from https://github.com/jumanjihouse/pre-commit-hook-yamlfmt/blob/master/setup.py#L13

```
    # globally-unique name
    packages=[
        'jumanjihouse_precommit_yamlfmt',
    ],
```